src/status_im/contexts/communities/overview/style.cljs

```diff
@@ -18,6 +18,7 @@
    :background-color background-color
    :height           (+ 20 ;; Area hidden by sheet on top but visible with rounded borders
                         92
+                        12
                         (safe-area/get-top))})
```

Comment:
Is this `12` part of the safe-areas special spacing? If so could we define a constant or function for retrieving this value?

Reply:
this `12` is just the padding of the navigation bar in any screen (but particularly on the communities overview). it must be counted to define the space to scroll.

Previously it wasn't counted because this screen was the only one that followed the expected safe areas, now with the safe-areas fixed, this component counts it.